src/components/10-atoms/input-text/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
@raphaellueckl add `navigator` to `eslintrc` as global instead of disabling a rule for the entire file!